### PR TITLE
modify background.

### PR DESCRIPTION
**Pull request by akshat157**
Modifies the background. its interactive now!

### CODE REVIEW

File: layout/styles/layout.css [hunk 1]
```diff
@@ -20,16 +20,27 @@ body, html {
...
-	z-index: 99;
-	background-color: rgba(255,255,255,0.44)
-	background-image: cover;+ height: 100%; background-attachment: fixed; margin: 0;;
+	//background-image:url('../../images/demo/backgrounds/04.jpg');
+	background-color: rgb(5, 5, 5);
+	background-size: 100% 100%; background-repeat: no-repeat; background-attachment: fixed; margin: 0;
 }
 .topspacer{padding-top: 50px;}
 
+/* particles-js */
```
Comment (shreyasbapat) [inline layout/styles/layout.css]: Awesome :)

**Comment by akshat157** [comment]
The change has been currently done only on the homepage. preview is available here.
https://akshat157.github.io/STAC-IITMandi.github.io/

**Comment by shreyasbapat** [comment]
Never mind! Make the appropriate changes in next PR.

**Comment by akshat157** [comment]
Yes. I know it. changes will be done in steps. may feel "incomplete" in the intermediate steps. don't mind those. :)

**Pull request merged**